JSDemos/Demos/Lookup/Templates/description.md

```diff
@@ -1 +1,9 @@
-This demo shows how you can customize the input field and the items of&nbsp;the Lookup component.
\ No newline at end of file
+The Lookup contains properties to specify custom templates for the following elements:
+
+* Input field: [fieldTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)
+
+* Items: [itemTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)
+
+* Group captions: [groupTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#groupTemplate)
+
+To customize items, you can also define [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) in the data objects. In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).
```

Comment:
>To customize items, you can also define [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) in the data objects.

Did I get it right and you suggest users to change data in the lookup's data source?

>In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).

As well as any other UI component, right?

Reply:
> [specific fields](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/)

The URL leads to a page titled "items", which is confusing because you didn't give the context.

> In addition, the Lookup supports [3rd-party template engines](/Documentation/Guide/UI_Components/Common/Templates/#3rd-Party_Template_Engines).

This feature works only with jQuery, and even jQuery users seldom use it. Let's remove this sentence.

```suggestion
To customize items, you can also define specific fields in data objects. Visit the [items](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) help topic for a full list of such fields. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) help topic for more information about these feature.
```